README.md

```diff
@@ -33,11 +33,85 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 - [List of hosted mainnets, testnets, and additional chains using BlockScout](https://docs.blockscout.com/for-projects/supported-projects)
 - [Hosted instance versions](https://docs.blockscout.com/about/use-cases/hosted-blockscout)
 
-
 ## Getting Started
 
+1. Install requirements
+
+    For a complete list of requirements, see the [blockscout docs](https://docs.blockscout.com/for-developers/information-and-settings/requirements).
+     > Note that we use older versions of Elixir and Erlang (see `.tool-versions`).  For help installing and managing these versions, you can follow the instructions in this article [here](https://medium.com/juq/how-to-manage-elixir-versions-on-mac-or-linux-getting-started-with-elixir-12308e7b6451).
+
+2. Set up some default configuration
+
+    ```shell
+    cp apps/explorer/config/dev.secret.exs.example apps/explorer/config/dev.secret.exs
+    cp apps/block_scout_web/config/dev.secret.exs.example apps/block_scout_web/config/dev.secret.exs
+    ```
+
+3. Add Env Variables
+
+4. Install Deps and Compile
+
+    ```shell
+    mix local.hex --force
+    mix local.rebar --force
+    mix deps.get
+    cd apps/block_scout_web/assets/ && \
+      npm install && \
+      npm run deploy && \
+      cd -
+    cd apps/explorer/ && \
+      npm install && \
+      cd -
+    mix compile
+    ```
+
+5. Start blockscout in a docker container
```

Comment:
thoughts on adding a 

"if you dont have docker installed get it [here](dockerlink)" 

Part?

Reply:
Added!